src/components/Form.test.js

```diff
@@ -13,191 +13,175 @@ describe('Form', () => {
     })
 
     it('submits with button click', () => {
-        const setUserPlace = jest.fn()
+        const startSearch = jest.fn()
```

Comment:
In order to indicate that ``startSearch`` is a mock function, you can name it ``startSearchMock``